ROMFS/px4fmu_common/mixers-sitl/rover_ackermann_sitl.main.mix

```diff
@@ -43,4 +43,4 @@ Z:
 
 Output 8
 ---------------------------------------
-Z:
+Z:
```

Comment:
Missing EOL

Reply:
odd, I just did an mv for that one. Fixed anyhow. Give me a sec to add the other file back and commit.